### ISSUE TITLE
Increase width of permissions dropdown (in Share menu)

### ISSUE THREAD

**Issue by jadeqwang** [sandstorm-io/sandstorm#877]
The permissions down down in the Share menu is a little bit too narrow for most things that will be displayed in the dropdown. This results in cropping, and in some cases, illegibility. 

![image](https://cloud.githubusercontent.com/assets/1584904/9857858/07311738-5ad2-11e5-901a-9a4fb478c4c7.png)

I think we could probably 1.5x-2x the width of that element without the left column feeling squeezed or anything.

Thoughts, @neynah?

**Comment by neynah** [comment]
I agree, it would also help to have the x aligned to touch the padding on the right.

**Comment by zarvox** [comment]
I'm wondering if perhaps this should have a different layout on mobile, since you have much less space to play with there.

**Comment by neynah** [comment]
+1 on looking into that.

**Comment by ocdtrekkie** [comment]
Can we throw a bite size label on this to make it easy to find? On my browser, it literally just shows "CAN" which doesn't tell me _anything useful at all_.
